src/components/RecipeList.tsx

```
@@ -168,7 +168,7 @@ const RecipeList: React.FC<RecipeListProps> = ({ recipes }) => {
                                     <span>{recipe.cook_time} min cuisson</span>
                                 </div>
                             )}
-                            {recipe.instructions}
+                            <div dangerouslySetInnerHTML={{ __html: recipe.instructions }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)